idd/Energy+.idd.in

```diff
@@ -6252,9 +6252,12 @@ WindowMaterial:Blind:EquivalentLayer,
   N4 , \field Slat Angle
        \type real
        \units deg
-       \minimum 0
-       \maximum 180
+       \minimum -90
+       \maximum 90
        \default 45
+       \note Slat angle is +ve if the tip of the slat front face is tilted upward, else  
+       \note the slat angle is -ve if the tip of the slat front face is tilted downward. 
+       \note The slat angle varies between -90 to +90. The default value is 45 degrees.
```

Comment:
OK.  So this needs a transition rule to subtract 90 degrees from the original input?